challengeOneWeek3.js

```diff
@@ -12,4 +12,21 @@
     // return numbers;
 
 
-   
\ No newline at end of file
+    function evenAndOddNumbers(num){
+        for(i=0; i<num.length; i++){
+            if(num[i]%2===0){
+                even.push(num[i])
```

Comment:
Where is the even variable defined?
Always avoid using global variables.
> Global variables are variables defined outside functions. They are at times hard to deal with when programs become complex since they can be changed by different parts of the program which might result in bugs that are hard to track.